src/main/kotlin/graph/DirectedGraph.kt

```diff
@@ -12,6 +13,15 @@ class DirectedGraph<V> : Graph<V>() {
         adjacencyList[firstVertexInd].add(Edge(secondVertexInd, label, weight))
     }
 
+    override fun getShortestPath(
+        start: V,
+        end: V,
+    ): MutableList<V>? {
+        val algo = BellmanFordAlgorithm(this)
+        if (vertexIndicesMap[end] == null || vertexIndicesMap[end] == null) throw IllegalArgumentException("Vertices can not be null")
```

Comment:
I think we shoud have another way for getting indices too. It's necessary for isAbleToAdd == false case.

Reply:
We can use our vertexValues ArrayList for that, cause we need to find only two indices.